editor/plugins/visual_shader_editor_plugin.cpp

```diff
@@ -413,32 +418,49 @@ void VisualShaderGraphPlugin::add_node(VisualShader::Type p_type, int p_id, bool
 	Ref<Theme> vstheme;
 	vstheme.instantiate();
 	Ref<Font> label_font = EditorNode::get_singleton()->get_editor_theme()->get_font("main_msdf", "EditorFonts");
+	Ref<Font> label_bold_font = EditorNode::get_singleton()->get_editor_theme()->get_font("main_bold_msdf", "EditorFonts");
 	vstheme->set_font("font", "Label", label_font);
+	vstheme->set_font("font", "GraphNodeTitleLabel", label_bold_font);
 	vstheme->set_font("font", "LineEdit", label_font);
 	vstheme->set_font("font", "Button", label_font);
```

Comment:
Not important right now, but worth extracting this into `editor_themes.cpp`. There isn't a good framework right now for creating additional themes in there, so that is going to require a bit of work.